cli/command/stack/list.go

```diff
@@ -31,5 +31,7 @@ func newListCommand(dockerCli command.Cli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.StringVar(&opts.Format, "format", "", "Pretty-print stacks using a Go template")
+	flags.BoolVarP(&opts.AllNamespaces, "all-namespaces", "", false, "List stacks among all Kubernetes namespaces")
```

Comment:
Wonder if we considered

- using `--namespace=all` (e.g.) to show all namespaces, also allowing listing stack in a single namespace `--namespace=<something>`
- showing all namespaces by default, but use a `--filter namespace=foo` to limit the output

Reply:
I believe what drives the current design is to mimic what ```kubectl``` does in order to not confuse Kubernetes users.